src/elements/container/sticky-bar/sticky-bar.visual.spec.ts

```diff
@@ -56,7 +57,7 @@ describe(`sbb-sticky-bar`, () => {
   `;
 
   describeViewports({ viewports: ['zero', 'medium', 'ultra'] }, () => {
-    describeEach(cases, ({ color, containerExpanded, scrolled }) => {
+    describeEach(cases, ({ color, containerExpanded, scrolled, size }) => {
```

Comment:
It doesn't really hurt, but I think we could reduce the amount of screenshot drastically by only testing size s in a separate test case. as it only is the block padding so far, every other param here doesn't have any effect. what do you think?